src/style.css

```diff
@@ -0,0 +1,153 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+
+  background: gray;
+
+  font-family: Roboto, sans-serif;
+}
+
+h2 {
+  margin: 0;
+}
+
+button,
+button:active,
+button:focus {
+  outline: none;
+  border: none;
+
+  background-color: transparent;
+}
+
+.card {
+  width: 200px;
+  height: 403px;
+  padding: 14px;
+  border-radius: 5px;
+  display: flex;
+  flex-direction: column;
+  border: 1px solid #f3f3f3;
+  box-shadow: 0 2px rgba(0, 0, 0, 0.07);
+
+  background-color: #fff;
+}
+
+.card__image {
+  width: 160px;
+  height: 134px;
+  align-self: center;
+  margin-top: 16px;
+}
+
+.card__info > h2 {
+  width: 100%;
+  margin-top: 45px;
+
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 15px;
+}
+
+.card__code {
+  width: 100%;
+  display: flex;
+  justify-content: flex-start;
+  align-items: flex-start;
+  margin-top: 5px;
+
+  font-size: 11px;
+  font-weight: 400;
+}
+
+.testimonials {
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-top: 15px;
+}
+
+.stars {
+  display: flex;
+  justify-self: start;
+}
+
+.stars__item {
+  width: 15px;
+  height: 15px;
+  margin-right: 4px;
+
+  background: url("/images/star.svg");
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars--1 .stars__item:nth-child(1),
+.stars--2 .stars__item:nth-child(-n+2),
+.stars--3 .stars__item:nth-child(-n+3),
+.stars--4 .stars__item:nth-child(-n+4),
+.stars--5 .stars__item:nth-child(n) {
+  background: url("/images/star-active.svg");
+  background-repeat: no-repeat;
+  background-position: center;
```

Comment:
If you fix the rules for all the stars as I wrote earlier, you can remove background-repeat and position here.